assets/js/user-form.js

```diff
@@ -2,13 +2,21 @@
 const $ = require('jquery');
 
 $(document).ready(function() {
-    $('.js-otherOccupation').parent().hide();
+    var $selectedOccupation = $('.js-occupation .form-check-input:checked');
+    var $occupationBlock =$('.js-occupation');
+    if ($selectedOccupation.length > 0) {
+        if ('-' === $selectedOccupation.val()) {
+            $occupationBlock.find('.form-control').parent().show();
```

Comment:
You cn maybe do sotheming like `$occupationBlock.find('.form-control').closest('SELECTOR').toggle('-' === $selectedOccupation.val());`